Release notes for Red Hat Developer Hub 1.4- Update Release Note Texts (#758)
redhat-developer/red-hat-developers-documentation-rhdh: modules/release-notes/snip-known-issue-rhidp-3396.adoc

[id="known-issue-rhidp-3396"]

> = Topology plugin permission does not show up in RBAC frontend UI

Reviewer: ```suggestion
= Topology plugin permission is not displayed in the RBAC front-end UI
```

> Some permissions that only have frontend plugins will not show up in the UI due to a requirement of needing a backend plugin to expose the permission framework's wellknow endpoint. As a workaround, you can either apply these permissions through the use of the CSV file or by directly calling the REST API of the RBAC backend plugin. Known plugins affected are Topology topology.view.read, Tekton tekton.view.read, ArgoCD argocd.view.read, and Quay quay.view.read.

Reviewer: ```suggestion
Permissions associated only with front-end plugins do not appear in the UI because they require a backend plugin to expose the permission framework's well-known endpoint. As a workaround, you can apply these permissions by using a CSV file or directly calling the REST API of the RBAC backend plugin. Affected plugins include Topology (`topology.view.read`), Tekton (`tekton.view.read`), ArgoCD (`argocd.view.read`), and Quay (`quay.view.read`).
```
